src/changelog.ts

```diff
@@ -164,24 +164,19 @@ function getTagUrl(repoUrl: string, tag: string) {
  * @param repoUrl - The URL for the GitHub repository.
  * @param tagPrefix - The prefix used in tags before the version number.
  * @param releases - The releases to generate link definitions for.
+ * @param [versionBeforePkgRename] - A version string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
+ * @param [tagPrefixBeforePkgRename] - A tag prefix string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
  * @returns The stringified release link definitions.
  */
 function stringifyLinkReferenceDefinitions(
   repoUrl: string,
   tagPrefix: string,
   releases: ReleaseMetadata[],
+  versionBeforePkgRename?: string,
+  tagPrefixBeforePkgRename?: string,
 ) {
-  // A list of release versions in descending SemVer order
-  const descendingSemverVersions = releases
-    .map(({ version }) => version)
-    .sort((a: Version, b: Version) => {
-      return semver.gt(a, b) ? -1 : 1;
-    });
-  const latestSemverVersion = descendingSemverVersions[0];
-  // A list of release versions in chronological order
-  const chronologicalVersions = releases.map(({ version }) => version);
-  const hasReleases = chronologicalVersions.length > 0;
-
   // The "Unreleased" section represents all changes made since the *highest*
```

Comment:
This comment now feels slightly out of place. The code that finds the "_highest_ release, not the most recent release" is now inside of `getUnreleasedLinkReferenceDefinition`. What do you think about moving this comment there? (Perhaps inside the function, at the top, before everything.)